docs/book/src/user/quick-start.md

```diff
@@ -483,8 +488,8 @@ kubectl --namespace=default get secret/capi-quickstart-kubeconfig -o jsonpath={.
 
 Calico is used here as an example.
 
-{{#tabs name:"tab-deploy-cni" tabs:"AWS|Docker|GCP|vSphere|OpenStack|Metal3,Azure"}}
-{{#tab AWS|Docker|GCP|vSphere|OpenStack|Metal3}}
+{{#tabs name:"tab-deploy-cni" tabs:"AWS|Docker|GCP|vSphere|OpenStack|Metal3,Azure,Packet"}}
```

Comment:
If packet uses calico like all the other providers (except Azure) this should be
```suggestion
{{#tabs name:"tab-deploy-cni" tabs:"AWS|Docker|GCP|vSphere|OpenStack|Metal3|Packet,Azure"}}
```